metaseq/trainer.py

```diff
@@ -774,9 +777,18 @@ def maybe_no_sync():
             ewm_loss_ratio = self.skip_spike(
                 logging_outputs, self.cfg.optimization.ewm_ratio_to_skip_batch
             )
+            # downscale grads by ewm_loss_ratio ** 4
+            if ewm_loss_ratio > 1.0:
```

Comment:
This means, that we are almost always downscaling the LR, should we not use a bit less aggressive threshold, so we only do the scaling when we reach an actual spike?

Reply:
I think our downscaling logic still isn't aggressive enough :( and technically we would probably be rolling back to previous checkpoints since it looks to still be too "delayed" when it gets triggered... leaving for now though to get bigbig up to date with what is currently running, and we can keep iterating :)